content/docs/hardware/general/case/_index.de.md

```diff
@@ -3,14 +3,224 @@ title: Gehäusedruck
++ Unter Umständen findet man in der Community hifreiche Bastler die einem ein Gehäuse mit herstellen. Am besten anfragen in Slack
++ In dieser Anleitung geht es darum ein komplettes Gehäuse mit Halterungen selbst zu drucken und herzustellen.
+Der Druck eines kompletten OBS Gehäusesatzes oder auch nur Teile ist problemlos möglich, vorrausgesetzt man hat einen 3D Drucker zur Verfügung. Der 3D-Druck kostet ca 5€ Material (Filamentkosten können variieren), Verschleiß, Stromkosten und Zeit. Dazu aber ca. 10-20h Druckzeit.
+Die nachfolgende Beschreibung baut auf dem [VerticalCase_JSN-AJ](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/tree/master/MainCase/VerticalCase_JSN-AJ) für das Sensorgehäuse auf und auf dem [SlideOnDisplay](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/tree/master/DisplayCase/SlideOnDisplay) für den Lenker Auslöseknopf und Anzeige. Aus Menge der angebotenen Modelle sind das die beiden Standardmodelle.  
+
+## Drucker, Material, SW, Druckeinstellungen 
+### Drucker und Material - PETG
+{{< imgproc "Ultimaker_at_work.jpg" Resize 400x >}} 
+Ultimkaer 3 fertig mit Displaygehäuse und Zugentlastung {{< /imgproc >}}
+Für den Druck verwende ich einen Ultimaker 3 dieser hat 2 extruder so das man mit 2 Filamenten gleichzeitig drucken, das wurde nur relevant für das OBS Logo und das ADFC logo, ansonsten ist das Gehäuse mit einem Filament gedruckt. Man kann nei Drucker mit einem Extruder natürlich das Filament wechseln.
+
+Wichtig ist die Slicer SW die den Druck für den jeweiligen Durcker aufbereitet. Für den Ultimaker (und viele andere Drucker) kann Cura https://ultimaker.com/software/ultimaker-cura verwendet werden (für Linux, Windows und Apple). Der Vorteil ist,  Cura kann man sich auf einen separaten PC laden und auf diesem offline probieren was die besten Einstellungen sind.  
+
+### Material und Farben
+
+PETG ist das Material das die meisten Vorteile für unser Projekt bietet:
+
++ hohe Schlagfestigkeit und dennoch flexibel - wichtig für den Einsatz am Fahrrad
++ Hydrophob, geringe Wasseraufnahme - wichtig für Außeneinsatz
++ niedrige Viskosität (Zähigkeit)
++ geruchsneutrales Drucken - erleichtert das Drucken auch zuHause in den 4 Wänden.
++ gute Witterungsbeständigkeit - Wichtig für Montage am Fahrrad
+- nachteilig ist allerdings die UV Beständigkeit die nicht so hoch ist wie bei ABS oder ASA. Doch die Vorteile überwiegen diese Einschränkuzng
+
+PETG ermöglicht auch den Kauf vieler verschieden farbiger und auch transparenter Filamente 
+{{< imgproc "OBS_main_case_and_lid_with_logos.jpg" Resize 800x >}} 
+Beispiel eines gedruckten Gehäuses mit Logos {{< /imgproc >}}
+
+## Druckvorbereitung und Druckdateien auswahl
+Alle Dateien für das Openbikesensor projekt wie SW, FW, Docs und auch die 3D Druckdateien liegen auf Github. Unter "OpenBikeSensor3dPrintableCase/MainCase/VerticalCase_JSN-AJ" sind viele verschiedene Druckdateien für unsere Gehäuse vorhanden
```

Comment:
Tippfehler
> … das Open**B**ike**S**ensor **P**rojekt …

Abkürzungen besser vermeiden
> … wie Software, Firmware, Dokumentation und ...

Besser einen klickbaren Link angeben
> … unter [OpenBikeSensor3dPrintableCase/MainCase/VerticalCase_JSN-AJ](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/tree/master/MainCase/VerticalCase_JSN-AJ) sind …

Reply:
Tippfehler, Abkürzung korrigiert
der link ist bewußt genommen, so dass jemand in Github auch den den Folder sieht und erkennt